### PR TITLE
Duluth wip

### PR DESCRIPTION
**Pull request by pdeutsch**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -48,7 +48,7 @@ task wrapper(type: Wrapper) {
     gradleVersion = '4.4'
 }
 
-wpi {
-    wpilibVersion = '2018.3.3'
-    ctreVersion = '5.2.2.0'
-}
+//wpi {
```
Comment (andrewfhou) [inline build.gradle]: Delete

File: properties/drivetrain.properties.competition [hunk 2]
```diff
@@ -3,14 +3,14 @@ TICKS_PER_INCH = 47.73
 DRIVETRAIN_WIDTH = 24
 MAX_VELOCITY_TICKS_PER_100MS = 750
 
-p_L = 0.035
-i_L = 0.007
+p_L = 2
```
Comment (pdeutsch) [inline properties/drivetrain.properties.competition]: Remove left and right - just have one set of PID values

File: src/main/java/frc/team1816/robot/Controls.java [hunk 3]
```diff
@@ -45,10 +45,10 @@ public Controls() {
...
         gamepad1.dPadRight().whenPressed(new SetElevatorHeightPercentCommand(60));
 
-        gamepad1.leftBumper().whenPressed(new RaiseCollectorClawCommand());
-        gamepad1.leftBumper().whenReleased(new StopCollectorClawCommand());
-        gamepad1.rightBumper().whenPressed(new LowerCollectorClawCommand());
-        gamepad1.rightBumper().whenReleased(new StopCollectorClawCommand());
+//        gamepad1.leftBumper().whenPressed(new RaiseCollectorClawCommand());
+//        gamepad1.leftBumper().whenReleased(new StopCollectorClawCommand());
+//        gamepad1.rightBumper().whenPressed(new LowerCollectorClawCommand(true));
+//        gamepad1.rightBumper().whenReleased(new StopCollectorClawCommand());
```
Comment (andrewfhou) [inline src/main/java/frc/team1816/robot/Controls.java]: Delete

**Review by andrewfhou — APPROVED** [review]
Everything looks good to me

**Pull request merged**